tests/bdd/features/lti_certification/section_3_types_of_roles.feature

```diff
@@ -84,8 +84,7 @@ Feature: Section 3 - Types of User Roles
 
      When I make an LTI launch request
 
-     Then the assigment opens successfully
-      And the user only has learner privileges
+     Then the response status code is 403
```

Comment:
I can't comment on that line but in line 78 the expected result is:


```
# Expected result: User should be given privileges appropriate to a user
# with no context role, which could be that they are denied access
```

switching now to "denied access" instead of our current "fallback to student".